packages/graph/hash_graph/lib/graph/src/types/schema/object.rs

```diff
@@ -2,7 +2,7 @@ use std::collections::HashMap;
 
 use serde::{Deserialize, Serialize};
 
-use crate::types::schema::{Uri, ValidationError};
+use crate::types::{schema::ValidationError, BaseUri};
```

Comment:
Might be fixed when I submit review, but this `BaseUri` doesn't exist in the types module (perhaps a rename operation didn't get through to `uri.rs`? related to https://github.com/hashintel/hash/pull/790/files#r919900649)

Reply:
I think this is fixed now?